code/game/objects/items/storage/bible.dm

```diff
@@ -18,26 +19,27 @@
 	desc = "To be applied to the head repeatedly."
 	icon_state ="bible"
 
-/obj/item/storage/bible/booze/Initialize(mapload, ...)
-	. = ..()
-	new /obj/item/reagent_containers/food/drinks/cans/beer(src)
-	new /obj/item/reagent_containers/food/drinks/cans/beer(src)
-	new /obj/item/spacecash(src)
-	new /obj/item/spacecash(src)
-	new /obj/item/spacecash(src)
+/obj/item/storage/bible/booze
+	spawns_with = list(
+		/obj/item/reagent_containers/food/drinks/cans/beer,
+		/obj/item/reagent_containers/food/drinks/cans/beer,
+		/obj/item/spacecash,
+		/obj/item/spacecash,
+		/obj/item/spacecash,
+	)
 
-/obj/item/storage/bible/afterattack(atom/A, mob/user, proximity)
+/obj/item/storage/bible/afterattack(atom/clicked_on, mob/user, proximity)
```

Comment:
same as before, make sure proc definition has same, should probably do in another PR